### DISCUSSION TITLE
Does it support email function?

### DISCUSSION THREAD

**Discussion by Hourunze1997** [FreshRSS/FreshRSS#7098] [Q&A] [answered]
I want to push messages from my feed to a certain mailbox at a certain frequency. Is this feature supported?

**Comment by Frenzie** [comment]
I don't believe so.

**Reply by Hourunze1997** [reply]
ok，thx

**Comment by Alkarex** [comment, marked as answer]
Yes, using a two-step process:

1. Generate an RSS feed from a FreshRSS user-query, filtering on what you are interested in https://freshrss.github.io/FreshRSS/en/users/user_queries.html#share-your-user-queries
2. Use whatever RSS-to-notification service, such as [IFTTT](https://ifttt.com/feed) or [alternatives](https://alternativeto.net/software/ifttt/?license=opensource&platform=self-hosted).

Note that some e-mail clients such as Thunderbird and Outlook can directly consume your RSS feed.

Related:
* https://github.com/FreshRSS/FreshRSS/issues/1718
* https://github.com/FreshRSS/FreshRSS/issues/2051

**Reply by Hourunze1997** [reply]
ok，thanks for your answer.

**Reply by Hourunze1997** [reply]
Can I use Share by email from the extension repository?

**Reply by math-GH** [reply]
The ShareByEmail extension adds one sharing service, so that you can share manually a selected article.
I think that it will not solve your wished workflow
